package.json

```diff
@@ -37,8 +37,10 @@
     "typescript": "^4.5.3"
   },
   "dependencies": {
+    "@gelatonetwork/constants": "^0.0.0",
```

Comment:
this one is a private package so we cannot use it in the public sdk

Reply:
[6d83c43](https://github.com/gelatodigital/ops-sdk/pull/25/commits/6d83c4358b6b909a31e19d3107a9c6d470990db6)